packages/accounts-app/src/Staking/Bond.tsx

```diff
@@ -62,8 +62,8 @@ export function Bond (props: Props) {
     stash = history.location.state.stash;
   }
 
-  const _validate = (): Either<Errors, AllExtrinsicData> => {
-    let errors: Errors = [];
+  const _validate = useCallback((): Either<Errors, AllExtrinsicData> => {
```

Comment:
just to be clear, using the `useCallback` hook will make it so the `_validate` function is only called when `[api, bond, controller, controllerBalance, derivedBalanceFees, destination, nonce, stash, stashBalance]` have updated right?

Reply:
Before:
```javascript
const _validate = () =>{}
useEffect(() => {}, [_validate, ...])
```

Which meant, on _each render_, the effect was called.

With usecallback:
```javascript
// When there's a change in [api, bond, controller, controllerBalance, derivedBalanceFees, destination, nonce, stash, stashBalance], create a new one
const _validate = useCallback(() =>{});
// Not called on each render
useEffect(() => {}, [_validate, ...])
```

Reply:
got it